controllers/actions.github.com/ephemeralrunnerset_controller.go

```diff
@@ -349,17 +349,14 @@ func (r *EphemeralRunnerSetReconciler) createProxySecret(ctx context.Context, ep
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      proxyEphemeralRunnerSetSecretName(ephemeralRunnerSet),
 			Namespace: ephemeralRunnerSet.Namespace,
-			Labels:    map[string]string{
-				// TODO: figure out autoScalingRunnerSet name and set it as a label for this secret
-				// "auto-scaling-runner-set-namespace": ephemeralRunnerSet.Namespace,
-				// "auto-scaling-runner-set-name": ephemeralRunnerSet.Name,
+			Labels: map[string]string{
+				"auto-scaling-runner-set-namespace": ephemeralRunnerSet.Namespace,
+				"auto-scaling-runner-set-name":      ephemeralRunnerSet.Name,
```

Comment:
is this label right? `auto-scaling-runner-set-name` is set to an `ephemeral-runner-set-name`

Reply:
🤦 that's why it was a todo, we have another issue to deal with labels to enable us to group all resources belonging to a scale set, reverting the todo.

Reply:
👍